core/drivers/crypto/caam/include/caam_acipher.h

```diff
@@ -49,4 +51,19 @@ static inline enum caam_status caam_ecc_init(vaddr_t ctrl_addr __unused)
 	return CAAM_NO_ERROR;
 }
 #endif /* CFG_NXP_CAAM_ECC_DRV */
+
+#ifdef CFG_NXP_CAAM_DSA_DRV
+/*
+ * Initialize the DSA module
+ *
+ * @caam_jrcfg   CAAM job ring configuration
+ */
+enum caam_status caam_dsa_init(struct caam_jrcfg *caam_jrcfg);
+#else
+static inline enum caam_status
+caam_dsa_init(struct caam_jrcfg *caam_jrcfg __unused)
```

Comment:
minor: would prefer return type next the function label, I think. debatable.
```c
static inline 
enum caam_status caam_dsa_init(struct caam_jrcfg *caam_jrcfg __unused)

Reply:
If you don't mind, I will keep the current formatting.

Reply:
ok